packages/sdk/src/internal/merge.ts

```
@@ -0,0 +1,160 @@
+
+export function deepMerge<T extends SchemaType>(
+    target: MergedModels<T>,
+    source: Partial<MergedModels<T>>
+): MergedModels<T> {
+    if (isCairoOption(target) && isCairoOption(source)) {
+        return mergeCairoOption(target, source);
+    }
+    if (isCairoCustomEnum(target) && isCairoCustomEnum(source)) {
+        return mergeCairoCustomEnum(target, source);
+    }
+    const result = { ...target } as Record<string, any>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)